rascal-vscode-extension/src/test/vscode-suite/utils.ts

```diff
@@ -269,11 +269,11 @@ export class IDEOperations {
 
 async function showRascalOutput(bbp: BottomBarPanel, channel: string) {
     const outputView = await bbp.openOutputView();
-    await outputView.selectChannel(`${channel} Language Server`);
+    await outputView.selectChannel(channel);
     return outputView;
 }
 
-export function printRascalOutputOnFailure(channel: 'Language Parametric Rascal' | 'Rascal MPL') {
+export function printRascalOutputOnFailure(channel: 'Language Parametric Rascal Language Server' | 'Rascal MPL') {
```

Comment:
I think the merge went wrong here. See https://github.com/usethesource/rascal-language-servers/commit/897c33d2d7c486a093d0c18e0752c1428c5cc982

as line 272 should not be changed, or otherwise the `Rascal MPL` also has to be postfixed with `Language Server`.